src/dynd/types/typevar_type.cpp

```diff
@@ -154,18 +154,16 @@ bool ndt::is_valid_typevar_name(const char *begin, const char *end)
   }
 }
 
-nd::array ndt::make_typevar_range(const char *name, intptr_t count)
+const std::vector<ndt::type> ndt::make_typevar_range(const char *name, intptr_t count)
 {
```

Comment:
Should be std::vector<ndt::type>, I don't think const on return makes sense. (Later on, we'll move this to `&&` anyway.)